Anime-App/Feature/Anime/View/AnimeViewController.swift

```diff
@@ -129,6 +138,42 @@ extension AnimeViewController {
         
     }
     
+    func fetchAnime() {
+        self.spinner.startAnimating()
+        
+        animeViewModel.fetchAnime(page: page) { [weak self] res in
+            switch res {
+            case .success:
+                self?.tableView.reloadData()
+                self?.isLoading = false
+            case .failure:
+                print("no more data")
+             }
+            self?.spinner.stopAnimating()
+        }
+    }
+    
+}
+
+
+extension AnimeViewController: UITableViewDataSourcePrefetching {
+
+    func tableView(_ tableView: UITableView, prefetchRowsAt indexPaths: [IndexPath]) {
+        for indexPath in indexPaths {
+            let lastItem = animeViewModel.animes.count - 1
+            if indexPath.row == lastItem {
+                page+=2
+                fetchAnime()
+            } else if page > pageLimit {
```

Comment:
When do you have this? This else if block is literally doing nothing. So can't you remove it? You only add an else if you have code that will fire in it. You don't to add it if you're not doing anything.